libyul/CompilabilityChecker.cpp

```diff
@@ -63,7 +63,9 @@ CompilabilityChecker::CompilabilityChecker(
 
 		for (StackTooDeepError const& error: transform.stackErrors())
 		{
-			unreachableVariables[error.functionName].emplace(error.variable);
+			auto& unreachables = unreachableVariables[error.functionName];
+			if (!util::contains(unreachables, error.variable))
+				unreachables.emplace_back(error.variable);
```

Comment:
Why not just keep it like?
```suggestion
			if (!util::contains(unreachableVariables[error.functionName], error.variable))
				 unreachableVariables[error.functionName].emplace_back(error.variable);
```

Or change `m_callGraph.functionCalls[m_currentFunction` [here](https://github.com/ethereum/solidity/pull/14311/files#diff-ec1c837efa56f7137e8dbaf9a3a46f863a284492c1b2c199ecea64bf6ea797d5L82) to a temporary variable as well?

Reply:
Technically, each such lookup has logarithmic cost in the size of the container, so (purely technically) the local variable makes it faster. Feel free to do it in the other case as well (even though there won't be a noticable difference in practice either way, most probably)